phpseclib/Crypt/Common/SymmetricKey.php

```diff
@@ -926,6 +932,16 @@ public function encrypt($plaintext)
                     }
 
                     return $ciphertext;
+                case self::MODE_CFB8:
+                    $ciphertext = openssl_encrypt($plaintext, $this->cipher_name_openssl, $this->key, OPENSSL_RAW_DATA | OPENSSL_ZERO_PADDING, $this->encryptIV);
+                    if ($this->continuousBuffer) {
+                            if (($len = strlen($ciphertext)) >= $this->block_size) {
+                                    $this->encryptIV = substr($ciphertext, -$this->block_size);
+                            } else {
+                                    $this->encryptIV = substr($this->encryptIV, $len - $this->block_size) . substr($ciphertext, -$len);
+                            }
```

Comment:
Quoting [PSR-2: Coding Style Guide](http://www.php-fig.org/psr/psr-2/), "_code MUST use 4 spaces for indenting, not tabs_". This piece of code is using 8 spaces for indenting - not 4.

Reply:
What the hell!! I didn't notice that...
I will fix it!

Should I exec `git rebase -i` to squash the commit and force update the PR?